src/libs/auth.js

```diff
@@ -196,3 +197,15 @@ authStore.subscribe((state, oldState) => {
     refreshTerraProfile().catch(error => reportError('Error loading user profile', error))
   }
 })
+
+authStore.subscribe(withErrorReporting('Error loading NIH account link status', async (state, oldState) => {
+  if (!oldState.isSignedIn && state.isSignedIn) {
+    try {
+      const nihStatus = await Ajax().User.getNihStatus()
+      authStore.update(state => ({ ...state, nihStatus }))
+    } catch (error) {
+      if (error.status === 404) authStore.update(state => ({ ...state, nihStatus: {} }))
```

Comment:
I'm surprised eslint was ok with the inline if/else construction, but for consistency could you bracket and linebreak these?